File: task3.js
let num = 101;
let start = 2;
let output = 'Sade ededdir';


if (num != 1) {
   while (start < num) {
      if (num % start == 0) {
         output = "Murekkeb ededdir";
         break;
      }
      start++;
   }

} else {
   output = "Ne murekkeb ne sade ededdir";
}

console.log(output);

File: task3.cs

using System;

public class HelloWorld
{
   public static void Main(string[] args)
   {
      int num = 1;
      int start = 2;
      String output = "Sade ededdir";


      if (num != 1)
      {
         while (start < num)
         {
            if (num % start == 0)
            {
               output = "Murekkeb ededdir";
               break;
            }
            start++;
         }

      }
      else
      {
         output = "Ne murekkeb ne sade ededdir";
      }

      System.Console.WriteLine(output);
      Console.ReadKey();
   }
}